src/server/utils/tokenizer.ts

```diff
@@ -1,13 +1,11 @@
 import lunr from "lunr";
-import nodejieba from "nodejieba";
+import jieba from "@node-rs/jieba";
 import { MatchMetadata } from "../../shared/interfaces";
 import { cutWordByUnderscore } from "./cutWordByUnderscore";
 
 // https://zhuanlan.zhihu.com/p/33335629
 const RegExpConsecutiveWord = /\w+|\p{Unified_Ideograph}+/u;
 
-nodejieba.load();
```

Comment:
Don't we need to call `jieba.load()` according to [its document](https://github.com/napi-rs/node-rs/tree/main/packages/jieba#usage)?

Reply:
not according to it's document, but I figure out in code, see:

https://github.com/napi-rs/node-rs/blob/3d329ba0ec01daabe4b9f6b1f2757d19d30be1c8/packages/jieba/src/lib.rs#L55

I encounter a test error "Jieba was loaded, could not load again" after I add the `load()`, but the error disappears after I remove the `load`;